cadc-test-vos/src/main/java/org/opencadc/conformance/vos/VOSTest.java

```diff
@@ -124,6 +124,7 @@ public abstract class VOSTest {
     public final URL synctransServiceURL;
     public final URL transferServiceURL;
     public final URL recursiveDeleteServiceURL;
+    public final URL pkgServiceURL;
```

Comment:
Is there a good reason that subclass-specific endpoints are accumulating here? For this one in particular, there are no pkg-specific methods in the base class... I would think the supporet methods here would only need the nodes endpoint.

Probably something I didn't pay attention to from earlier work,  but it looks more and more like poor style.

Reply:
A couple are common to multiple tests, nodes and synctrans. Subsequent tests probably just followed the same pattern. I forgot to remove pkgServiceURL, it's not used. Moved the others to the tests where they are used.